docs/proposals/multi-version.md

```diff
@@ -0,0 +1,53 @@
+# Multi-Version-Landscaper-Instances
+
+This document contains a proposal how to support multiple versions of Landscaper Instances in one Landscaper as a 
+Service (LaaS) landscape.
+
+## Requirements
+
+- A LaaS landscape should support different versions of a Landscaper instance (LI), i.e. different LandscaperDeployments 
+  install different versions of a [this](../../.landscaper/landscaper-instance) component. A more detailed
+  selection of particular deployer versions seems to be too complicated and out of scope.
+
+- A LaaS landscape supports a predefined set of LI versions, which change from release to realease (probably if the LaaS 
+  component version changes).
+
+- The Central Landscaper should always be on the latest Landscaper version. The version referenced by the 
+  LaaS project.  
+
+- A customer must be able 
+  - to select its intended LI version during onboarding. This must be optional. He gets the latest version
+    if nothing is selected.
+  - to upgrade to its intended LI version. Therefore, an allowed upgrade path must be enforced.
+  - see which versions are currently supported, which are recommended and which are outdated and what are the possible
+    upgrade paths.
+
+- How to upgrade a customer LI instance if his current version is not supported anymore? This might become even more 
+  critical if a security issue came up.
+  - Proposal for an initial solution: 
+    - Not supported versions are not upgraded. Only the reconciliation of the corresponding Installation is retriggered
+      for token rotation issues, or we enhance the Installation by some automatic reconcile mechanisms.
+    - Affected customers are informed via email. This mail is send automatically and if the customer does not upgrade,
+      we get informed to personally contact him.
```

Comment:
I would not like to have any manual steps here like contacting stakeholders personally. This might scale initially, but will not in the future. Automatic emails is ok, but if stakeholders do not upgrade, shouldn't we just force-upgrade ? For sure this has to be mentioned super prominently in our LaaS documentation, so that stakeholders know of this force-upgrade procedure if they don't react.

Reply:
let discuss this later

Reply:
adapted aas discussed